### DISCUSSION TITLE
Missing autogenerated tiles

### DISCUSSION THREAD

**Discussion by PiotrMachowski** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#160] [FAQ]
If you use `vacuum_platform: default` and your card is missing tiles with sensors' statuses you have to enable these sensors in settings of miio integration in integrations page.

[![Open your Home Assistant instance and show your integrations.](https://my.home-assistant.io/badges/integrations.svg)](https://my.home-assistant.io/redirect/integrations/)

**Comment by c1tru55** [comment]
@PiotrMachowski 
what if there is no such setting?
![](https://i.imgur.com/bM7zj2N.png)
the only setting is to use cloud to get list of connected devices

**Reply by PiotrMachowski** [reply]
You have to click "16 entities", then enable every disabled entity that you want to have in card

**Reply by Cupmar** [reply]
Hi I enabled all entities but still the tiles and icons do not appear in card. Any further idea?

**Reply by PiotrMachowski** [reply]
@Cupmar have you refreshed the page? Which vacuum do you have?

**Reply by Cupmar** [reply]
Yes i did even a reboot. My Version is a Roborock S50 - the interesting part is that Miio integration shows only 13 entities and not 16
P.S. i migrated my config from an earlier version

**Reply by PiotrMachowski** [reply]
Have you selected "default" vacuum platform? 
If entity for a specific tile is missing then it is not card's fault.
